emaki.rb

```diff
@@ -222,6 +241,37 @@
   redirect to '/'
 end
 
+delete '/slides/:id' do
+  @slide = Slide.first(id: params[:id])
+  unless @slide
+    status 404
+    return slim :"attentions/slide_not_found", layout: :layout
+  end
+  @user = @slide.user
+  unless @user.slug == session[:user]
+    status 403
+    attention :only_named_user
+    redirect to '/'
+    return
+  end
+
+  # スライドの実態ファイル削除
+  unless Binder.rmdir(@user.slug, @slide.slug)
+    status 500
+    attention :slide_delete_error
+    redirect to "/users/#{@user.slug}"
+  end
+
+  if @slide.destroy
+    attention :slide_deleted
+    redirect to "/users/#{@user.slug}"
+  else
+    status 500
+    attention :slide_delete_error
+    redirect to "/users/#{@user.slug}"
```

Comment:
__[rubocop]__
* Style/IdenticalConditionalBranches: Move `redirect to "/users/#{@user.slug}"` out of the conditional.

<!-- Comment by [SideCI](https://sideci.com) -->